pygeoapi/provider/oracle.py

```diff
@@ -33,6 +34,8 @@
 import oracledb
```

Comment:
Can we update the import ordering as follows:

```python
import importlib
import json
import logging
import os
import threading
from typing import Optional

import oracledb
import pyproj

from pygeoapi.api import DEFAULT_STORAGE_CRS
```

Reply:
Changed accordingly